benchmarking/chain-sync/prepare-byron-proxy.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -e
+
+BASEDIR=$(realpath $(dirname $0))
+#GITURL="https://github.com/input-output-hk/cardano-byron-proxy.git"
+TARGETDIR=cardano-byron-proxy.git
+
+#rm -rf ${TARGETDIR}
+#git clone -b master ${GITURL} ${TARGETDIR}
+git submodule update --init --remote ${TARGETDIR}
```

Comment:
In https://github.com/input-output-hk/cardano-node/pull/269 we'll get a different way of pinning the proxy, so we'd have to change this -- but that's good for now.